example/conf/manifest/init

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+from cdist_manifestation.dependencies import order_dependency, require
+from cdist_manifestation.types import file, directory
+
+with order_dependency():
```

Comment:
This example does not make much sense since only one object is created here

Reply:
Fixed.

Reply:
Example changed to be more sensible.